cmd/kubeadm/app/constants/constants.go

```diff
@@ -210,10 +210,13 @@ const (
 	// CertificateKeySize specifies the size of the key used to encrypt certificates on uploadcerts phase
 	CertificateKeySize = 32
 
-	// LabelNodeRoleMaster specifies that a node is a control-plane
-	// This is a duplicate definition of the constant in pkg/controller/service/controller.go
+	// LabelNodeRoleMaster specifies that a Node hosts control-plane components
```

Comment:
clarifying the note, since a control-plane can include many components from different nodes.